kickstarts/almalinux-8-micro.x86_64.ks

```diff
@@ -22,41 +22,30 @@ shutdown
 %packages --ignoremissing --excludedocs --instLangs=en --nocore --excludeWeakdeps
 coreutils-single
 glibc-minimal-langpack
-
--binutils
--brotli
--dnf
--findutils
 -crypto-policies-scripts
--diffutils
--elfutils-debuginfod-client
--firewalld
--gettext*
--glibc-langpack-en
+-dosfstools
+-e2fsprogs
+-fuse-libs
 -gnupg2-smime
--grub\*
--hostname
--iptables
--iputils
--langpacks-en
 -kernel
--libevent
--less
--openssl
--os-prober
+-langpacks-en
+-libss
 -open-vm-tools
 -pinentry
--platform-python-pip
+-qemu-guest-agent
 -shared-mime-info
--tar
 -trousers
--unbound-libs
--vim-minimal
+-xfsprogs
 -xkeyboard-config
--xz
--yum
 %end
 
+%pre
+# Pre configure tasks for Docker
+
+# Don't add the anaconda build logs to the image
+# see /usr/share/anaconda/post-scripts/99-copy-logs.ks
+touch /tmp/NOSAVE_LOGS
```

Comment:
Just a question, is this requried for other types like minimal, init etc?

Reply:
I believe yes (btw, nice catch @LKHN). People don't need installer logs in containers and we can reduce image size.